recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - jasper
     - libpng >=1.6.23,<1.7
     - libnetcdf 4.4.*
-    - perl
+    - perl 5.22.2.1
```

Comment:
Update the libpng pin too `>=1.6.28,<1.7` (and, since that is a functional change, bump the build number.)